src/test/java/com/google/api/codegen/testdata/python_smoke_test_library.baseline

```diff
@@ -0,0 +1,42 @@
+============== file: test/google/cloud/gapic/example/library/v1/library_service_smoke_test.py ==============
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Smoke tests."""
+
+import argparse
+import time
+import unittest
+
+from google.cloud.gapic.example.library.v1 import enums
+from google.cloud.gapic.example.library.v1 import library_service_client
+from google.cloud.proto.example.library.v1 import field_mask_pb2 as v1_field_mask_pb2
+from google.cloud.proto.example.library.v1 import library_pb2
+from google.protobuf import field_mask_pb2 as protobuf_field_mask_pb2
+
+
+class LibraryServiceSmokeTest(unittest.TestCase):
+
+    def test_update_book(self):
+        parser = argparse.ArgumentParser(description='Verify the build with a smoke test.')
+        parser.add_argument('project_id', metavar='PROJECT-ID', nargs='1', help='Id of your test project', required=True)
+        args = parser.parse_args()
+        project_id = args.project_id
+
+        client = library_service_client.LibraryServiceClient()
+        name = client.book_path("testShelf-" + time.time(), project_id)
+        rating = enums.Book.Rating.GOOD
+        book = library_pb2.Book(rating=rating)
+        response = client.update_book(name, book)
+        print(response)
```

Comment:
In Java we print a 'OK' as a signal that the test ends so if we run multiple tests together we can easily distinguish each test. It would be nice for Python does the same.